rock-papper-scissors/main.py

```
@@ -1,27 +1,27 @@
         
-
+#list of options program uses
 options = ["Rock", "Paper", "Scissor"]
 
 while True:
     UI = input("\nEnter Your Choice (rock, paper, scissor) : ").title()
     
     if UI not in options:
-        continue
+        continue #program continues
     print(winner(UI, random.choice(options)))
```

Ruff (0.15.6)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)